utilities.py

```diff
@@ -0,0 +1,35 @@
+def flat(data, parent_dir):
+    '''
+    Convert JSON data to a dictionary.
+    
+    :param list/dict data : json data.
+    :param list parent_dir: json fields. 
+    
+    :return dict: {subtest:json_value}
+    '''
+    def recursive_helper(data, parent_dir, ret):

```

Comment:
Generally, we put `_` in front of the function names if they are nested inside other functions.